intel_extension_for_transformers/neural_chat/examples/serving/TGI/README.md

```diff
@@ -63,7 +63,6 @@ You can customize the configuration file 'tgi.yaml' to match your environment se
 | model_name_or_path                | "./neural-chat-7b-v3-1"                 |
```

Comment:
better to make the readme more complete. After launching the service, how end users can interact with the service?

Reply:
The example of comsumption of services are added to the README.